src/libxively/event_dispatcher/xi_time_event.c

```diff
@@ -0,0 +1,348 @@
+/* Copyright (c) 2003-2015, LogMeIn, Inc. All rights reserved.
```

Comment:
Copyright dates and BSD 3-clause disclaimer isn't up to date here .Check other sources.